### ISSUE TITLE
Unexpected keyword error occurs in lc_wrapper/kernel.py

### ISSUE THREAD

**Issue by yo-saito** [NII-cloud-operation/Jupyter-LC_wrapper#59]
Hi there

I installed the LC_wrapper according to "How to Install", but I get the following error (infinite loop) and it does not work.
Is there a solution?

### Error
```
ERROR | get_msg() got an unexpected keyword argument 'block
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/dist-packages/lc_wrapper/kernel.py", line 90, in run
    msg = self.channel.get_msg(block=True, timeout=0.2)
TypeError: get_msg() got an unexpected keyword argument 'block'
```

### Procedure

1. run # jupyterhub -f /etc/jupyterhub/jupyterhub_config.py
2. access JupyterHub from a web browser and open the LC_wrapper kernel.
3. keep getting an error on the terminal where JupyterHub is started.
4. The LC_wrapper kernel will be able to input data, but will not respond with execution results, and the IPython execution status will remain [*].

### Environment

- Ubuntu 18.04 LTS

```
# jupyterhub --version
1.4.2
# jupyter notebook --version
6.4.3
# python3 --version
Python 3.6.9
```

**Comment by yacchin1205** [comment]
Thanks for the report! @scsk-saito 

This problem seems to be caused by the change https://github.com/jupyter/jupyter_client/pull/671 since jupyter-client 7.0.0.
We will fix this issue. If you need it immediately, you may be able to get it to work by downgrading the jupyter-client version.

**Comment by scsk-saito** [comment]
@yacchin1205 
Thank you for your confirmation.
I downgraded the jupyter-client version from 7.0.2 to 6.2.0, but it recurred.
I would appreciate it if you could fix it.

**Comment by yacchin1205** [comment]
@yo-saito Fixed the problem. jupyter-client library also needs a fix https://github.com/jupyter/jupyter_client/issues/696 (released as jupyter_client 7.0.4), so please upgrade both lc_wrapper and jupyter_client. Thank you 😄
